### DISCUSSION TITLE
annotation of traces in langfuse console?

### DISCUSSION THREAD

**Discussion by erik-squared** [langfuse/langfuse#1760] [Ideas]
### Describe the feature or potential improvement

is there any interest in being able to annotate a trace manually via the web interface? 

use-case would be performing a root cause on a low-performing query and centralizing the capture of the analysis.

**Comment by marcklingen** [comment]
you can add manual feedback via the `add score` button today. we're planning to extend this feature a lot over the next weeks to be able to score on multiple dimensions

**Comment by marcklingen** [comment]
@erik-squared this might be an interesting addition here: https://langfuse.com/changelog/2024-10-10-annotation-queues

**Reply by erik-squared** [reply]
checking it out!
